test/e2e/resize_nodes.go

```diff
@@ -372,18 +410,19 @@ var _ = Describe("Nodes", func() {
 					break
 				}
 			}
-			Logf("Setting network partition on %s", node.Name)
+			By(fmt.Sprintf("cause network partition on node %s", node.Name))
+
 			dropCmd := fmt.Sprintf("sudo iptables -I OUTPUT 1 -d %s -j DROP", testContext.CloudConfig.MasterName)
```

Comment:
Note also that only outbound packets from the node will be dropped.  You might also want to drop inbound packets from the master.